engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala

```diff
@@ -23,6 +45,34 @@ case class EngineMetadataServiceActor(serviceConfig: Config, globalConfig: Confi
     }
   }
 
+  private def queryStatusAndRespond(id: WorkflowId): Unit = {
+    val sndr = sender()
+    dataAccess.getWorkflowStatus(id) onComplete {
```

Comment:
I'm not 100% sure I love this arrangement actually.  Now workflows might be submitted and a UUID returned, but a  status poll will return a 404 until the first summary sweep.